andela/travel_tool_front
# #162375478 Fix approval page link access

#### What does this PR do?
This PR limits the access of the approvals page

#### Description of Task to be completed?
- Ensure the approvals page is only available to the managers and super admin.

#### How should this be manually tested?
* Clone the repos: ``` https://github.com/andela/travel_tool_front ``` and ``` https://github.com/andela/travel_tool_back ```.
* cd into their respective directories ```travel_tool_front``` and ```travel_tool_back```.
* On the front end checkout `ft-fix-approval-link-access-162375478` branch.
* Start the backend by running the `$yarn run start:dev`
* Login and you should not be able to view the approval page link on the left sidebar if your role is a `Requester`, `Travel Team Member` or `Travel administrator`.
* Change your role to either a `Manager` or a `Super Admin` you should have access to the `Approvals` page through the Approvals link on the left sidebar.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162375478](https://www.pivotaltracker.com/story/show/162375478)

#### Screenshots (if appropriate)


#### Questions:
